components/markdown/src/markdown.rs

```diff
@@ -679,7 +679,7 @@ pub fn markdown_to_html(
                         event
                     });
                 }
-                Event::Html(text) => {
+                Event::Html(text) | Event::InlineHtml(text) => {
```

Comment:
Sadly it's not _that_ easy. There is https://github.com/getzola/zola/pull/2581 which solves both (I did miss it solved the shortcode issue sorry :() but I still don't know whether the continue reading in inline html makes sense.
To keep the fix simple, it should probably be a separate branch that doesn't contain the continue reading stuff and we can decide later if we want the full #2581

Reply:
You're more the welcome to copy my code and remove the continue-reading bit, FWIW. (Aimed at both amtanq/Keats if either of you want to do it first.)

Otherwise, I'll get around to it at some point.

Reply:
Since you have more context on it, would appreciate if you can pick this whenever you get time. LMK if you're okay with it, else would be happy to continue myself too!